src/main/java/org/tron/common/runtime/vm/CallCreate.java

```diff
@@ -23,32 +23,32 @@
  */
 public class CallCreate {
 
-    final byte[] data;
-    final byte[] destination;
-    final byte[] gasLimit;
-    final byte[] value;
-
-
-    public CallCreate(byte[] data, byte[] destination, byte[] gasLimit, byte[] value) {
-        this.data = data;
-        this.destination = destination;
-        this.gasLimit = gasLimit;
-        this.value = value;
-    }
-
-    public byte[] getData() {
-        return data;
-    }
-
-    public byte[] getDestination() {
-        return destination;
-    }
-
-    public byte[] getGasLimit() {
-        return gasLimit;
-    }
-
-    public byte[] getValue() {
-        return value;
-    }
+  private final byte[] data;
+  private final byte[] destination;
+  private final byte[] energyLimit;
+  private final byte[] value;
+
+
+  public CallCreate(byte[] data, byte[] destination, byte[] energyLimit, byte[] value) {
+    this.data = data;
+    this.destination = destination;
+    this.energyLimit = energyLimit;
+    this.value = value;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Store a copy of "value". ([squid:S2384](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2384))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1324&issues=AWVbdRtysniGm95Wz7EK&open=AWVbdRtysniGm95Wz7EK)

[ISSUE_KEY]: AWVbdRtysniGm95Wz7EK